modules/monitoring/library/Monitoring/Command/Renderer/IcingaApiCommandRenderer.php

```diff
@@ -99,6 +99,9 @@ public function renderAddComment(AddCommentCommand $command)
             'author'    => $command->getAuthor(),
             'comment'   => $command->getComment()
         );
+        if ($command->getExpireTime() !== null) {
+            $data['expiry'] = $command->getExpireTime();
+        }
```

Comment:
Please add an empty line after this block. We require an empty line after closing braces.